test/hpy_devel/test_distutils.py

```diff
@@ -90,7 +90,7 @@ def python(self, *args, capture=False):
             proc = subprocess.run(cmd)
             out = None
         if proc.returncode != 0:
-            raise Exception(f"Command {exe} failed")
+            raise Exception(f"Command {cmd} failed")
```

Comment:
This is a fix for something not connected to the PR. I don't understand why these tests are failing.

Reply:
Fixed in master.